### PR TITLE
media-sound/spotify: version bump to 1.0.66

### PR DESCRIPTION
**Pull request by alexbakker**

### CODE REVIEW

**Comment by prometheanfire** [comment]
1.0.64 is still around, can you change this to a copy and modify instead of a move?

File: media-sound/spotify/spotify-1.0.66.ebuild [hunk 1]
```diff
@@ -6,10 +6,10 @@ inherit eutils fdo-mime gnome2-utils pax-utils unpacker
...
-BUILD_ID="407.g9bd02c2d"
+BUILD_ID="478.g1296534d"
 SRC_BASE="http://repository.spotify.com/pool/non-free/s/${PN}-client/"
-SRC_URI="amd64? ( ${SRC_BASE}${PN}-client_${PV}.${BUILD_ID}-26_amd64.deb )
-	x86? ( ${SRC_BASE}${PN}-client_${PV}.${BUILD_ID}-26_i386.deb )"
+SRC_URI="amd64? ( ${SRC_BASE}${PN}-client_${PV}.${BUILD_ID}-39_amd64.deb )
+	x86? ( ${SRC_BASE}${PN}-client_${PV}.${BUILD_ID}-39_i386.deb )"
 LICENSE="Spotify"
 SLOT="0"
 KEYWORDS="amd64 x86"
```
Comment (a17r) [inline media-sound/spotify/spotify-1.0.66.ebuild]: You can't add a new version with stable keywords.

**Comment by alexbakker** [comment]
Fixed. I incorrectly assumed that spotify removed 1.0.64 from their repository.

**Comment by prometheanfire** [comment]
looks like the manifest is broken?

**Comment by alexbakker** [comment]
Sigh. Third time's a charm.

**Comment by prometheanfire** [comment]
merged

**Pull request closed without merging**